### PR TITLE
760 release prep

### PR DESCRIPTION
**Pull request by chiatt**
### Description of Change
Prepares 7.6.0 for release - updates version number, formats release notes

### CODE REVIEW

File: releases/7.6.0.md [hunk 1]
```diff
@@ -29,65 +29,66 @@ Arches 7.6.0 Release Notes
...
+- Fix workbench sidepanel header misalignment occuring during provisional edit review [#11323](https://github.com/archesproject/arches/issues/11323)
+- Add project-level testing and GitHub test runners [#10842](https://github.com/archesproject/arches/issues/10842)
+- Lenient file-type checking mode [#10862](https://github.com/archesproject/arches/issues/10862)
+- Allow overriding search_results view [#10699](https://github.com/archesproject/arches/issues/10699)
+- Styling fix in resource model manage menu [#10911](https://github.com/archesproject/arches/issues/10911)
+- Harden SystemSettings model against too-early access [#11118 [#11118]](https://github.com/archesproject/arches/issues/1118])
+- Upgrade openpyxl package to 3.1.2 and fixes ETL modules [#10726](https://github.com/archesproject/arches/issues/10726)
+- Implement arches version compatibility check as a Django system check [#11114](https://github.com/archesproject/arches/issues/11114)
+- Add unlocalized string datatype [#9191](https://github.com/archesproject/arches/issues/9191)
```
Comment (jacobtylerwalls) [inline releases/7.6.0.md:62]: I might promote this one up to the highlights.

File: releases/7.6.0.md [hunk 2]
```diff
@@ -546,6 +538,8 @@ Minor incompatibilities:
     {% endblock arches_translations %}
     ```
 
+13. Run `python manage.py migrate`.
+
 14. Within your project, with your Python 3 virtual environment activated:
```
Comment (jacobtylerwalls) [inline releases/7.6.0.md:543]: Activating the environment should happen at least as early as migrate.

File: releases/7.6.0.md [hunk 3]
```diff
@@ -29,65 +29,66 @@ Arches 7.6.0 Release Notes
 - Adds permission frameworks - including "standard" (current behavior and default) and "default deny" (future (8.x) default - all users are denied resource permissions until they are explicitly granted). This also gives developers the ability to implement their own custom permission framework.
 
 ### Performance Improvements
-- 10453 Reduce queries for related objects when indexing resources
+- Reduce queries for related objects when indexing resources [#10453](https://github.com/archesproject/arches/issues/10453)
+- Add index to the edit_log table's resourceinstanceid column to improve performance [#11499 ](https://github.com/archesproject/arches/issues/1499 )
```
Comment (jacobtylerwalls) [inline releases/7.6.0.md:33]:
```suggestion
- Add index to the edit_log table's resourceinstanceid column to improve performance [#11499](https://github.com/archesproject/arches/issues/14999)
```

File: releases/7.6.0.md [hunk 4]
```diff
@@ -29,65 +29,66 @@ Arches 7.6.0 Release Notes
...
+- Update Google Analytics syntax supporting Google Analytics 4 [#11132](https://github.com/archesproject/arches/issues/11132)
+- Filter out tiles created during resource creation from activity stream API [#9768](https://github.com/archesproject/arches/issues/9768)
+- Ensure resource creation edit log timestamps precede resource update timestamps [#9769](https://github.com/archesproject/arches/issues/9769)
+- Support Shapefile import in map editor widget [#11339](https://github.com/archesproject/arches/issues/11339)
+- Fix nuisance failing requests to help template view [#11478](https://github.com/archesproject/arches/issues/11478)
+- Adds Github action for comparing test coverage between branches and rejecting branches that lower test coverage [#10738](https://github.com/archesproject/arches/issues/10738)
+- Ensure search facets have widget label on advanced search restore [#11046](https://github.com/archesproject/arches/issues/11046)
+- Persist unsaved value in string widget if a user switches between cards in the card tree [#11092 ](https://github.com/archesproject/arches/issues/1092 )
```
Comment (jacobtylerwalls) [inline releases/7.6.0.md:53]:
```suggestion
- Persist unsaved value in string widget if a user switches between cards in the card tree [#11092](https://github.com/archesproject/arches/issues/11092)
```

File: releases/7.6.0.md (hunk 2 above)
Comment (chiatt) [reply releases/7.6.0.md:543]: Ah - good catch.

File: releases/7.6.0.md (hunk 1 above)
Comment (chiatt) [reply releases/7.6.0.md:62]: Agreed

**Review by jacobtylerwalls — APPROVED** [review]

**Pull request merged**